documentation/modules/cruise-control/proc-implementing-rebalance-proposal.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-implementing-rebalance-proposal-{context}']
+= Implementing a rebalance proposal
+
+If you are satisfied with a rebalance proposal, you can implement it by annotating the corresponding `KafkaClusterRebalance` resource. 
+
+At the next reconciliation, the Cluster Operator will instruct Cruise Control to rebalance the Kafka cluster. 
+Cruise Control then reassigns partitions to brokers as summarized in the rebalance proposal.
+
+WARNING: This is not a dry run. 
+Review the rebalance proposal before you implement it.
+
+.Prerequisites
+
+* You have xref:proc-generating-rebalance-proposals-{context}[generated a rebalance proposal].
+
+* The rebalance proposal is ready to implement (the `KafkaClusterRebalance` custom resource status is `ProposalReady`).
+
+.Procedure
+
+. (Optional) Refresh the rebalance proposal that you want to implement:
```

Comment:
we should be clear why the user should refresh so if after having the proposal ready, he doesn't execute it for long time and it becomes stale.